x-pack/test/detection_engine_api_integration/basic/tests/query_signals.ts

```diff
@@ -12,33 +12,22 @@ import {
   ALERTS_AS_DATA_FIND_URL,
 } from '@kbn/security-solution-plugin/common/constants';
 import { FtrProviderContext } from '../../common/ftr_provider_context';
-import { getSignalStatus, createSignalsIndex, deleteSignalsIndex } from '../../utils';
+import {
+  getSignalStatus,
+  createSignalsIndex,
+  clearSignalsIndex,
+  deleteSignalsIndex,
+} from '../../utils';
 
 // eslint-disable-next-line import/no-default-export
 export default ({ getService }: FtrProviderContext) => {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
   const log = getService('log');
+  const es = getService('es');
 
   describe('query_signals_route and find_alerts_route', () => {
     describe('validation checks', () => {
-      it('should not give errors when querying and the signals index does not exist yet', async () => {
```

Comment:
deleted this test because the index is installed on Kibana start instead of on first rule execution so it should always exist when the signals are queried.